### PR TITLE
L2 895 proposal card display consistency

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

To make the sns proposal card more consistent with nns its design ([L2-895](https://dfinity.atlassian.net/browse/L2-895)) was updated.

# Changes

- remove voting status
- update `ProposalMeta` component to show more info

# Tests

- fields in ProposalCard.spec
- update proposal meta specs

# Screenshots

image

image

image

### CODE REVIEW

File: frontend/src/lib/components/launchpad/ProposalCard.svelte [hunk 1]
```diff
@@ -21,13 +29,56 @@
   };
 </script>
 
-<Card role="link" withArrow on:click={showProposal} testId="sns-proposal-card">
+<Card>
   <h3 slot="start">{title}</h3>
-  <VotesProgress {proposalInfo} />
+
+  <ul>
```
Comment (peterpeterparker) [inline frontend/src/lib/components/launchpad/ProposalCard.svelte:35]: can we/you extract a component for `Proposer`, `id` and `deadline` and reuse it here but also on the nns list of proposals screen?

the idea is to display the same information and to add the `deadline` there too.

Comment (mstrasinskis) [reply frontend/src/lib/components/launchpad/ProposalCard.svelte]: Good point, done.

File: frontend/src/lib/components/proposals/ProposalMeta.svelte [hunk 2]
```diff
@@ -1,57 +1,82 @@
...
+  export let showUrl: boolean = false;
+  export let showTopic: boolean = false;
 
   let id: ProposalId | undefined;
   let topic: string | undefined;
   let url: string | undefined;
-
   $: ({ id, url, topic } = mapProposalInfo(proposalInfo));
+
+  let deadline: bigint | undefined;
```
Comment (peterpeterparker) [inline frontend/src/lib/components/proposals/ProposalMeta.svelte:18]: maybe we can add the parsing of the `deadline` in `mapProposalInfo` too or does not make sense?

File: frontend/src/lib/components/launchpad/ProposalCard.svelte [hunk 3]
```diff
@@ -5,14 +5,15 @@
...
+  import { i18n } from "../../stores/i18n";
+  import ProposalMeta from "../proposals/ProposalMeta.svelte";
 
   export let proposalInfo: ProposalInfo;
 
   let id: ProposalId | undefined;
   let title: string | undefined;
 
-  $: ({ id, title } = mapProposalInfo(proposalInfo));
+  $: ({ id, url, title } = mapProposalInfo(proposalInfo));
```
[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/src/lib/components/launchpad/ProposalCard.svelte]: Good catch!

File: frontend/src/lib/components/proposals/ProposalMeta.svelte (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/proposals/ProposalMeta.svelte]: Good point! Done

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**